spec/amber/controller/base_spec.cr

```diff
@@ -83,6 +83,39 @@ module Amber::Controller
         context.request.headers["Accept"] = "text/plain"
         ResponsesController.new(context).index.should eq expected_result
       end
+
```

Comment:
Can you add a test for an invalid (unsupported) extension?

Reply:
Yeah. I'll commit that in a second. I already have a test for a supported but undefined extension.

Reply:
Done.